qiskit/pulse/calibration_entries.py

```diff
@@ -34,11 +34,12 @@ class CalibrationEntry(metaclass=ABCMeta):
     """A metaclass of a calibration entry."""
 
     @abstractmethod
-    def define(self, definition: Any):
+    def define(self, definition: Any, user_provided: bool):
```

Comment:
No, all subclasses provide default value for `user_provided` to conform to previous default behavior. Currently in 0.23.x, this flag is implicitly determined by type information and only `PulseQobjDef` is recognized as a backend provided entry. However one may want to provide backend calibration as `ScheduleDef`, to bypass cumbersome JSON data generation. This could be a separate PR though.